src/forms/BoundForm.stories.tsx

```diff
@@ -221,6 +221,22 @@ const inputConfig: BoundFormInputConfig<AuthorInput> = [
     }),
   },
 
+  {
+    listFieldBooks: {
+      name: "Book",
+      rows: [{ title: boundTextField() }, { isPublished: boundSwitchField() }],
+      defaultValues: { isPublished: false, title: undefined },
```

Comment:
Thought: Maybe an `onNew` callback, that took the `objectState` as a prop, so could do dynamic things?

Reply:
oh yeah that would be nice